ee/tabby-webserver/src/repositories/resolve.rs

```diff
@@ -47,6 +47,15 @@ impl ResolveParams {
     pub fn path_str(&self) -> &str {
         self.path.as_deref().unwrap_or("")
     }
+
+    #[cfg(not(target_os = "windows"))]
+    pub fn os_path(&self) -> String {
+        self.path.clone().unwrap_or_default()
+    }
+    #[cfg(target_os = "windows")]
+    pub fn os_path(&self) -> String {
+        self.path.clone().unwrap_or_default().replace("/", r"\")
```

Comment:
is there a standard library function to handle such conversion?

Reply:
No such function from std AFAIK.

Actually, I was planning to use [path_slash](https://docs.rs/path-slash/latest/path_slash/) crate, but later thought that it wasn't worthwhile to import another dep for such simple case. But you can take a look at it and let me know your thoughts.

Reply:
sg. Then let's use `cfg! statement`

```rust
if cfg!(target_os != "windows") {
   ...
} else {
  ...
}
```